openverse_catalog/dags/common/requester.py

```diff
@@ -65,6 +75,12 @@ def get(self, url, params=None, **kwargs):
             # This exception will only be raised during testing, and it *must*
             # be re-raised and bubbled up the stack
             raise
+        except (AirflowException, KeyboardInterrupt):
+            # These exceptions are raised by task managers and should be respected &
+            # re-raised. Airflow runs its tasks in a separate thread, so if this
+            # exception is received, typically it means that the task has been
+            # sent a SIGTERM, which means that the task should be stopped.
+            raise
```

Comment:
TIL that you can re-raise the last error with simple `raise` statement with no arguments.

Reply:
Very handy for cases like this 😄 There's also [`raise ... from ...`](https://www.pythontutorial.net/python-oop/python-raise-from/), which can be really useful for making stack traces more specific!